tests/test_lib_usercache.py

```diff
@@ -105,6 +105,8 @@ def test_01_get_username_from_cache(self):
         self.assertEqual(u_name, "")
 
     def test_02_get_resolvers(self):
+        # enable user cache
```

Comment:
I think the changes in the test is also part of a fix?
I would also like to move this to another commit/PR.

Reply:
This is due to an update of the SQLALchemy package. The `UserCache` entry will be removed after the `self._delete_realm()` call so we can't access the data anymore.